### PR TITLE
Fixes issue #751, redundant code in nexus driver

### PR DESCRIPTION
**Pull request by SahilTikale**
Fixes issue #751.
Found redundant code in the nexus (switch) driver.

### CODE REVIEW

**Comment by SahilTikale** [comment]
@naved001 @shwsun please review it. 
small PR. We can merge it soon.

**Comment by naved001** [comment]
passed the deployment tests. so +1

**Comment by SahilTikale** [comment]
@shwsun  this is a small PR. Can you please some time to review it. 
It passes all deployment tests.

File: haas/ext/switches/nexus.py [hunk 1]
```diff
@@ -105,12 +105,6 @@ def connect(switch):
         console.expect('Password: ')
         console.sendline(switch.password)
 
-        console = pexpect.spawn('telnet ' + switch.hostname)
```
Comment (knikolla) [inline haas/ext/switches/nexus.py]: I would like to know more as to why this was here twice in the first place.

Comment (SahilTikale) [inline haas/ext/switches/nexus.py]: I came across the redundant code during my recent cloud computing mentor meeting.

Comment (zenhack) [inline haas/ext/switches/nexus.py]: Maybe a botched merge at some point? Could dig through the history to find out.

**Comment by zenhack** [comment]
I'd also like to know how this got here, but the patch looks good to me.

File: haas/ext/switches/nexus.py (hunk 1 above)
Comment (zenhack) [inline haas/ext/switches/nexus.py]: `git-blame` blames me: dfdc7f8. Apparently I goofed somehow way back and nobody noticed until now. +1, merging.

**Pull request merged**